package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@qooxdoo/framework",
-  "version": "7.9.1",
+  "version": "8.0.0-beta",
```

Comment:
I don't think this is right. The new version 8 class/property stuff isn't in this PR.

Reply:
his serves to prepare a v8_Beta branch by merging all pr marked with v8

Reply:
ok then!

Reply:
all tests are green now